### DISCUSSION TITLE
Size/File Count in Sources Does Not Update for Exclude If Present

### DISCUSSION THREAD

**Discussion by ksnell88** [borgbase/vorta#1783] [FAQ]
Hello,

I'm using BORG_NOBACKUPDIR.TAG in the Exclude If Present section on the Sources tab, and it appears to properly exclude the directories with the tag in it, so that is great.

What I did notice is that adding this tag to a directory and then clicking the "Refresh Source Size and File Count" does not update with the new expected size and file count.

Is this a bug or working as expected? 

I would have liked for it to show me how large it expects my backup to be from the directory after exclusions as the Repository has limited space. Perhaps even better would be a "Total Size" column and then a "Total After Exclusions" so it is easy to see the impact of one's exclusions?

![2023-08-21_vorta_exclude_dir-source_size](https://github.com/borgbase/vorta/assets/34407093/6a71afc0-fa42-4bcb-852b-8c6d7fa8abf5)

**Comment by real-yfprojects** [comment]
> Is this a bug or working as expected?

This is expected. See https://github.com/borgbase/vorta/discussions/1764#discussioncomment-6713678 for further insight.
